Services/DeviceService.m

```diff
@@ -260,6 +260,8 @@ - (DeviceServicePairingType) pairingType
     return DeviceServicePairingTypeNone;
 }
 
+- (void) setPairingType:(DeviceServicePairingType)pairingType { }
```

Comment:
Please leave a comment explaining why this setter is empty. Otherwise, someone may think it's an error.